### ISSUE TITLE
Long period of network disconnect breaks IPSEC

### ISSUE THREAD

**Issue by sheng-liang** [rancher/rancher#8688]
**Rancher Versions:**
1.5.8
Server:
healthcheck:
ipsec:
network-services:
scheduler:
kubernetes (if applicable):

**Docker Version:**

**OS and where are the hosts located? (cloud, bare metal, etc):**
VMware Workstation VM
**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
Single node
**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Cattle
**Steps to Reproduce:**
1. Create 3 VMs on VMware Workstation (or VirtualBox, or Fusion)
2. Setup single node Rancher on one host, and register the other 2 hosts.
3. Create an instance of the `shengliang/apache-php` container on each host
4. Inside one apache-php container, you can connect to the other apache-php container using this command:
    ```
    while true; do curl -sSf http://10.42.c.d -o /dev/null; echo `date` $?; done
    ```
    `10.42.c.d` is the IP of the other apache-php container.
5. Disconnect the network from the host running the first apache-php container. You can see the curl command starts to fail.
6. Wait for 4 hours.
7. Reconnect the network to the host running the first apache-php container.

**Results:** 
The first apache-php container still cannot connect to the second apache-php container.
Healthcheck container on the first host is broken too.
Restarting the IPSEC router (do `docker kill` the IPSEC router container) brings the network back to life.

**Comment by stszap** [comment]
Isn't it the same issue https://github.com/rancher/rancher/issues/9971 ?

**Comment by superseb** [comment]
Yes this should be fixed by #9971 , thanks
